cmd/cmd.go

```diff
@@ -5,9 +5,11 @@ import (
 	"os"
 )
 
-// Bail handles exitable errors in commands.
+// BailOnError handles exitable errors in commands.
 // TODO: figure out what goes here.
-func Bail(err error) {
-	log.Println(err)
-	os.Exit(1)
+func BailOnError(err error) {
+	if err != nil {
+		log.Println(err)
```

Comment:
Yeah, I don't feel strongly yet about what we put here, since it's just a placeholder. If you have some thoughts about how we should deal with errors, would you go ahead and submit a PR with them so we can discuss? (Or open an issue).

For this I just want to bring it in, as it's mostly about tucking the `if err != nil {}` bit into this method than getting the actual behavior right on exit.